tests/test_utils/test_schemaview.py

```diff
@@ -945,6 +947,34 @@ def test_is_inlined(self):
                 actual_result = sv.is_inlined(slot)
                 self.assertEqual(actual_result, expected_result)
 
+    def test_materialize_nonscalar_slot_usage(self):
```

Comment:
```suggestion
    def test_materialize_nonscalar_slot_usage(self):
        """
        ``slot_usage`` overrides values in the base slot definition without
        clobbering unrelated, nonscalar values.
        
        See: 
        - https://github.com/linkml/linkml/issues/2224
        - https://github.com/linkml/linkml-runtime/pull/335
        """
```

Just giving us a little hint for our future selves that can be skimmed :) am on phone, sorry if this is broken.

Reply:
Something about that docstring is not working with the code highlighting, thats what I get for reviewing from a phone lol